app/models/amusement_park.rb

```diff
@@ -1,3 +1,5 @@
 class AmusementPark < ApplicationRecord
   has_many :rides
+
+  validates :name, :admission_cost, presense: true
```

Comment:
For ICs, you're not required to set up validations. You can skip this next time in order to maximize time on user stories!